client/public/about.html

```diff
@@ -501,6 +501,17 @@ <h5 class="fw-bold">Camron Olsen</h5>
                 <p class="text-muted">Drexel Fall 2024</p>
               </div>
             </div>
+          </div> <div class="col-md-4">
```

Comment:
Please move the new div to the next line to maintain established formatting.

Reply:
Yeah sure